introduction.tex

```diff
@@ -1,3 +1,15 @@
+\section{Abstract}
+France is in pursuit of closing the fuel cycle by transitioning from a \gls{LWR} fleet
```

Comment:
Since this is somewhat in question since Macron, it is appropriate to refer to the CNE reports (particularly 2012 and later) which inform french policy https://www.cne2.fr/index.php/en/cne-2-2007-to-this-day. It may be most correct to state the facts as explicitly as possible (e.g. "The French strategy recommended by 2012-2015 Commission Nationale d'Evaluation reports \cite{cne_rapport_YYYY} emphasizes preparation for a transition from LWRs to SFRs.")